profiler/src/ProfilerEngine/Datadog.Profiler.Native/LinkedList.hpp

```diff
@@ -15,6 +15,7 @@ namespace pmr {
 using namespace std::experimental::pmr;
 }
 #else // Not found at all
+// cppcheck-suppress preprocessorErrorDirective
```

Comment:
Since CppCheck cannot find `<memory_resource>` header file, it defaults to `#error`.
Here we suppress the error because it's not one :)